sage_modabvar/lseries.py

```diff
@@ -171,6 +174,41 @@ def _repr_(self):
         """
         return "Complex L-series attached to %s"%self.abelian_variety()
 
+    def vanishes_at_1(self):
+        """
+        Return True if `L(1)=0` and return False otherwise.
+
+        OUTPUT:
+            a boolean
+
+        EXAMPLES::
+
+            sage: from sage_modabvar import J0, J1
+            sage: L = J0(389)[0].lseries(); L
+            Complex L-series attached to Simple abelian subvariety 389a(1,389) of dimension 1 of J0(389)
+            sage: L(1)
+            -1.33139759782370e-19
+            sage: L.vanishes_at_1()
+            True
+
```

Comment:
use words and explain what is going on here, namely -- heh it looks like zero but isn't as an algebraic computation shows.  Then compute L(1) to higher precision and see that it is still about 1e-7.